ops/demo/container_instances.tf

```diff
@@ -10,4 +10,19 @@ module "db_liquibase_action" {
   image_action            = var.image_action
   spring_datasource_url   = data.azurerm_key_vault_secret.sr_db_jdbc.value
   subnet_id               = data.terraform_remote_state.persistent_demo.outputs.subnet_container_instances_id
+}
+
+module "db_client" {
+  source                  = "../services/container_instances/db_client/infra"
+  count                   = 1
```

Comment:
Not blocking. Unless we see ourselves in a position where we don't want this module deployed, we probably don't need this `count`.

If we do imagine turning this off for some environments at some point, an alternative could be defining a variable `db_client_count` with the `0` or `1` value that we want for that environment. Setting up a variable would also let us override it at runtime.

Reply:
Great catch! I've removed this unneeded variable.